### ISSUE TITLE
Version 2.6.2 (e) – Unknown column 'c.brewerMHP' in 'field list'

### ISSUE THREAD

**Issue by 003401E8** [geoffhumphrey/brewcompetitiononlineentry#1479]
**Verify the Following**
Reporting About the Latest Version? **Yes** 

Searched Issue Reports for Bug? **Yes**

Reviewed the Good To Know Issue Reports? **Yes**

**BCOE&M Info**
Version: Version 2.6.2 (e)
Installation URL: https://opferm.cialers.org/index.php?section=past-winners&go=2023
Hosted Installation? Not sure

**Describe the Bug**
When viewing past winners (i.e. https://opferm.cialers.org/index.php?section=past-winners&go=2023) there is an error at the bottom of the page stating "Unknown column 'c.brewerMHP' in 'field list'"

**To Reproduce**
1. On a competition page that has past winner history available, click on the year and see the bottom of the page.

**Expected Behavior**
Expected behavior is to see the winners for all categories of that year instead of Best of Show only.

**Screenshots**
![image](https://github.com/geoffhumphrey/brewcompetitiononlineentry/assets/71900912/83854549-4f7d-4ccc-9fb7-e7e93794f320)

**Comment by geoffhumphrey** [comment]
If you do a force update, this field should be added to the brewer DB table. To do a force update, open your /paths.php file, and change the FORCE_UPDATE constant variable to TRUE. Once you've done that, refresh or access your root URL ONCE. Then, change the variable back to FALSE.

Line 110:

```
define('FORCE_UPDATE', TRUE);
```

If that doesn't work, you can manually add the brewerMHP field to the brewer DB table using this SQL command (if you're using a prefix, change xxx_ to your prefix - if not, remove the xx_ below):

```
ALTER TABLE `xxx_brewer` ADD `brewerMHP` int(11) NULL DEFAULT NULL;
```

Hope that helps!

**Comment by 003401E8** [comment]
I'm just an entrant into that competition, but I will forward this info to their competition coordinator. Thanks!

**Comment by geoffhumphrey** [comment]
This bug has been fixed in the latest 2.6.2 release (tag 2.6.2.1).
